libbeat/dashboards/import_dashboards.go

```diff
@@ -448,16 +448,15 @@ func (imp Importer) ImportDir(dirType string, dir string) error {
 	// check if the directory exists
 	if _, err := os.Stat(dir); err != nil {
 		// nothing to import
-		fmt.Println("No directory", dir)
-		return nil
+		return fmt.Errorf("No directory %s", dir)
```

Comment:
Actually, shouldn't passing a non-existent dir to this script be an error? I think the fix here is to make the test harness a bit smarter. It shouldn't blindly construct a path that doesn't exist and pass it to the `import_dashboards` code.